### DISCUSSION TITLE
Problem downloading zip files

### DISCUSSION THREAD

**Discussion by PtaxaDev** [community/community#49655] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Good night, I have vaznikovila here is the problem I know that this is unlikely to you, but maybe you can help me... Kind of a problem with certificates although if you look through them expires much later than the day on which the request took place... This error occurs when I'm feeding a zip file... anything. Thanks for understanding.:

Your connection is not closed
Attackers may try to steal your data from codeload.github.com (e.g. passwords, messages or credit cards).
NET::ERR_CERT_COMMON_NAME_INVALID
Codeload.github.com uses encryption to protect your information. The last time Microsoft Edge tried to connect to codeload.github.com, this website sent unusual and incorrect credentials. The reason may be an attempt by attackers to impersonate codeload.github.com or an interrupted connection on the Wi-Fi login screen. Your information is still protected because Microsoft Edge stopped the connection before the data exchange.

It is now impossible to open codeload.github.com because this website uses the HSTS mechanism. Usually network errors and attacks are temporary, so this page will probably become available after some time.

**Comment by Roccoz2004** [comment]
I am having this same issue, and it is making it impossible for scripts to install necessary utilities from GitHub repositories.

**Comment by rebootedtill** [comment]
Same issue here. For me it has to be something Local because I can download from my iPad/iPhone and even a VM on my PC. We’re you able to find a solution?

**Reply by FeN1X1945** [reply]
solved a problem?

**Comment by ParNishka14** [comment]
Yeah, just delete github.com from your host.txt, if u cant’n find google
host file windows. Good luck

Вт, 21 мая 2024 г. в 23:09, FeN1X1945 ***@***.***>:
